charmstore.go

```diff
@@ -232,6 +233,94 @@ func (s *CharmStore) Latest(curls ...*charm.URL) ([]CharmRevision, error) {
 	return responses, nil
 }
 
+func (s *CharmStore) ListResources(ids ...*charm.URL) ([]ResourceResult, error) {
```

Comment:
I think we have enough wiggle room to take a slice here.  Latest() isn't the best example.